src/sentry/api/endpoints/user_emails.py

```diff
@@ -171,6 +171,16 @@ def put(self, request, user):
         if has_new_username and not User.objects.filter(username__iexact=new_email).exists():
             update_kwargs["username"] = new_email
 
+        # NOTE(mattrobenolt): When changing your primary email address,
+        # we explicitly want to invalidate existing lost password hashes,
+        # so that in the event of a compromised inbox, an outstanding
+        # password hash can't be used to gain access. We also feel this
```

Comment:
```suggestion
        # password hash can be used to gain access. We also feel this
```

Reply:
That makes this incorrect. "When changing, we explicitly want, so that in the event, an outstanding password hash **can't** be used." Probably just a long sentence that's hard to read because I'm bad at writing, but it defintiely should be **can't**.

Reply:
oh nvm i misread this